classes/class-p4-master-site.php

```diff
@@ -202,12 +202,27 @@ public function set_featured_image( $post_id, $post, $update ) {
 	 * @param WP_Post $post_before Whether this is an existing post being updated or not.
 	 */
 	public function clean_post_cache( $post_id, $post_after, $post_before ) {
-
 		// Ignore autosave.
 		if ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) {
 			return;
 		}
-		clean_post_cache( $post_id );
+		// Lua Script.
+		$lua = <<<LUA
+local k =  0
+for i, name in ipairs(redis.call('KEYS', KEYS[1]))
+do
+    redis.call('DEL', name)
+    k = k+1
+end
+return k
+LUA;
+
+		$group = 'nginx-cache';
+		$redis = new Redis();
+		if ( $redis->connect( '127.0.0.1', 6379, 5 ) ) {
```

Comment:
@27Bslash6 I know this won't work for docker compose (since the redis container has other IP).
Any ideas on this one?

Reply:
The redis server is configured in multiple places: it's a global variable in wp-config.php ( https://github.com/greenpeace/planet4-docker/blob/master/src/planet-4-151612/wordpress/wp-config.php.tmpl#L83 ), as a database option `rt_wp_nginx_helper_options`, and according to https://github.com/greenpeace/planet4-nginx-helper/blob/feature/PLANET-3268-fix-server-host/admin/class-nginx-helper-admin.php#L305 it's defined as a constant.